### ISSUE TITLE
AVT 2 - DataTable w/Toolbar search placeholder does not meet color contrast requirements

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4225]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React
Color Contrast Analyzer

## Detailed Description
Go to the React DataTable w/Toolbar Component 
Run Color Contrast Analyzer and test the search placeholder text against the background color (see screenshot):
<img width="1434" alt="Screen Shot 2019-10-04 at 9 38 15 AM" src="https://user-images.githubusercontent.com/21676914/66217000-18f48a00-e68c-11e9-9b95-7dc8c583e139.png">
